src/Console/Application.php

```diff
@@ -46,11 +46,11 @@ public function doRun(InputInterface $input, OutputInterface $output): int
 
     public function getLongVersion(): string
     {
-        return trim(sprintf(
+        return sprintf(
```

Comment:
doesn't make sense to use trim as there are no spaces at the left or right sides of this string